### ISSUE TITLE
Cannot build android with latest gradle versions

### ISSUE THREAD

**Issue by sigma-libra** [enoosoft/moon_phase#2]
When building with gradle versions 8 and up, I get the following error:

```
FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring project ':moon_phase'.
> Could not create an instance of type com.android.build.api.variant.impl.LibraryVariantBuilderImpl.
   > Namespace not specified. Specify a namespace in the module's build file. See https://d.android.com/r/tools/upgrade-assistant/set-namespace for information about setting the namespace.

     If you've specified the package attribute in the source AndroidManifest.xml, you can use the AGP Upgrade Assistant to migrate to the namespace value in the build file. Refer to https://d.android.com/r/tools/upgrade-assistant/agp-upgrade-assistant for general information about using the AGP Upgrade Assistant.
```

**Comment by bksubhuti** [comment]
Yes.. I'm here for the same issue.
```
FAILURE: Build failed with an exception.

* What went wrong:
The Android Gradle plugin supports only Kotlin Gradle plugin version 1.5.20 and higher.
The following dependencies do not satisfy the required version:
project ':moon_phase' -> org.jetbrains.kotlin:kotlin-gradle-plugin:1.3.50
```

**Comment by bksubhuti** [comment]
oh.. it looks like your complaint is not soooo old.. but it looks like this is not so well supported . 
What I did was copy the code directly into my project.  I made a moon_phase directory and copied all of the the lib files there.  It worked fine.

If you want to help.. I'd like to have the moon be a picture of the moon and add black to cover that picture disk.

**Comment by sigma-libra** [comment]
Yeah, I ended up copying over the files, and that worked fine. For the colors, I will see what I can do.

**Comment by bksubhuti** [comment]
The centering was off.. I had to do an offset 

```
                child: Transform.translate(
                  offset: Offset(27,
                      27), // adjust the xOffset to reposition the moon to the left

                  child: MoonWidget(
                    size: 100,
                    moonColor: const Color.fromARGB(97, 63, 57, 57),
                    date: selectedDate,
                  ),
                ),

```

This moon is public domain (as far as I know unless previously copyrighted.. i declare it as cc0) created by dalle3 .. green background to be removed.  If you can overlay the painting on this moon.. this would be useful.  The green tint on the moon face is a trick of the eyes from the background.

![dalleMoon](https://github.com/enoosoft/moon_phase/assets/18184280/8d180351-c198-414b-8222-9f35860d80e7)

**Comment by bksubhuti** [comment]
this is also cc0 if it was not done already (by some other guy i don't know). 
![dalleMoon-2](https://github.com/enoosoft/moon_phase/assets/18184280/25de730d-a5d3-483d-aa2c-d239dcc7fa18)

**Comment by bksubhuti** [comment]
With some gpt+ stuff.  I was able to get the moon to work.  The painting is being done on 0,0 coordinates which is why it is not centered.

I have code for this I can do a PR but it seems nobody is responding.

![image](https://github.com/enoosoft/moon_phase/assets/18184280/fc051e86-5e98-4d25-a5b0-1a53ab858483)

**Comment by enoosoft** [comment]
@bksubhuti  

thanks for fixing this! I know I've been MIA on this repo. 
Go ahead with the PR - curious to see how you handled the coordinate issue.

Build issues are fixed now.

**Comment by bksubhuti** [comment]
@enoosoft  Hi.. sorry for the delay.. i saw it and then did not act on this.

Here are the files.. 

[moon_phase.zip](https://github.com/user-attachments/files/23694927/moon_phase.zip)
if you want to see how to use it.. look at the moon screen.
Download the app Buddhist Sun apple or android.
The repo is here.
https://github.com/bksubhuti/buddhist_sun

The moon picture is generated by gpt ai and then I altered it a little bit to give some better colors.
should be legit to use for free I presume.  I have another moon photo if you want to use it.

I'm not good at git..and doing PR's don't let my apps fool you.  I just use vsc GUI interface and when something goes wrong, i ask gpt to bail me out.  :)

I suggest you just copy my files and push direct to your repo.
